### PR TITLE
docs(tenants): document the tenants api

### PR DESCRIPTION
**Pull request by dcsiebe**
Add a file that documents the tenants api.

### CODE REVIEW

**Review by kvdb — APPROVED** [review]

File: docs/api/tenants.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+
+### URL
+
+```http
+GET /tenants HTTP/1.1
+```
+
+### Response
+
+The response contains a list of organisations.
```
Comment (joggienl) [inline docs/api/tenants.md]: It contains a list of organisations, or a list of tenants? I would say "tenants" here.

File: docs/api/tenants.md [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+
+Name         | Type     | Description
+-------------|----------|------------
+id           | `string` | **Optional** A unique identifier. If none is given, one is generated.
+name         | `string` | **Required** The name of the tenant.
+
+### Request
+
+```http
+POST /organisation HTTP/1.1
```
Comment (joggienl) [inline docs/api/tenants.md]: The post should also go to the /tenant URI ?

**Review by joggienl — CHANGES_REQUESTED** [review]
Overall OK.
- There is a branch conflict that has to be solved before the PR can be merged
- Some naming issues (organisation v.s. tenant)

File: docs/api/tenants.md (hunk 1 above)
Comment (dcsiebe) [reply docs/api/tenants.md]: done

File: docs/api/tenants.md (hunk 2 above)
Comment (dcsiebe) [reply docs/api/tenants.md]: done

**Pull request closed without merging**